.github/workflows/release.yml

```diff
@@ -115,6 +115,11 @@ jobs:
         with:
           macos-target: 15.0
           build-backend: ${{ matrix.python-version == '3.10' }}
+      - name: Build macOS 26 package
+        uses: ./.github/actions/build-macos-release
+        with:
+          macos-target: 26.0
+          build-backend: ${{ matrix.python-version == '3.10' }}
```

Comment:
We will do the 26.0 release manually for now until we can get on macOS 26.2 (which is needed to build neural accelerator and RDMA support).

So please remove that part.

Reply:
Yes I think it's great to start testing 26.0 👍 

For the nightly it's fine.. the only downside is burning compute.. but it's also not necessary so we can remove it and add it back once we do the 26.0 release in CI.